hushline/__init__.py

```diff
@@ -126,3 +146,43 @@ def add_onion_location_header(response: Response) -> Response:
                     app.logger.warning("HostOrganization data not found in database.")
 
     return app
+
+
+def register_commands(app: Flask) -> None:
+    stripe_cli = AppGroup("stripe")
+
+    @stripe_cli.command("configure")
+    def configure() -> None:
+        """Configure Stripe and premium tiers"""
+        # Make sure tiers exist
+        with app.app_context():
+            free_tier = db.session.query(Tier).filter_by(name="Free").first()
```

Comment:
We have several places where we use the strings `"Business"` and `"Free"` as well as their price. Perhaps these should be constants in the `Tier` model? Or perhaps since there's a few places create them, we have `@classmethod`s for `create_{free,business}_tier()`

Reply:
Fixed in 3757b5ec6137201111e00ed8335b73f8e5388721